src/main/scala/cromwell/Main.scala

```diff
@@ -36,19 +42,20 @@ object Main extends App {
    * Also now passing args to runAction instead of the constructor, as even sbt seemed to have issues with the args
```

Comment:
I know this wasn't you but the phrasing of this comment is weird in that it only makes sense in the context of whatever PR it came from